extensions/cornerstone/src/initWADOImageLoader.js

```diff
@@ -83,6 +79,34 @@ export default function initWADOImageLoader(
   initWebWorkers(appConfig);
 }
 
+const generateAcceptHeader = (configAcceptHeader = [], omitQuotationForMultipartRequest = false) => {
+  let acceptHeader = []
+  if (configAcceptHeader.length === 0) {
+    acceptHeader.push('multipart/related; type=application/octet-stream')
+  } else {
+    acceptHeader = configAcceptHeader
+  }
+
+  if (!omitQuotationForMultipartRequest) {
```

Comment:
The point of the configuration accept header is to provide a header as is, without needing any tweaks.  The point of providing a transfer syntax UID is that you can easily configure a single TSUID in the right type, and not need to worry about it - also, so that the TSUID can come from the query response, eg in an AvailableTransferSyntaxUID method.
So, if the config accept header  is provided, use it - either as a comma separated string, or as an array (I had forgotten you could do either), and if it is provided as a transfer syntax UID, then use the omit quotation for multipart request, and provide a single header response in that TSUID.

Reply:
Ok thanks, here is a new version